spec/models/order_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe Order, type: :model do
+  before(:all) { User.create(fullname: "David Daniel", 
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Trailing whitespace detected.